Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -9,6 +9,18 @@
 module Homebrew
   module_function
 
+  # Get versions for specific architectures
+  class NewVersion
```

Comment:
feels more like a https://ruby-doc.org/core-2.5.0/Struct.html or https://sorbet.org/docs/tstruct?

Reply:
I had some issues with T::Struct so I ended up moving some of the methods and checks in the class. I may end up reusing the class for the bump command, so I'm happy for any suggestions on how to improve it.